app/code/Magento/Integration/Controller/Adminhtml/Integration/Delete.php

```diff
@@ -44,7 +44,7 @@ public function execute()
                         $this->_oauthService->deleteConsumer($integrationData[Info::DATA_CONSUMER_ID]);
                     }
                     $this->_registry->register(self::REGISTRY_KEY_CURRENT_INTEGRATION, $integrationData);
-                    $this->messageManager->addSuccess(
+                    $this->messageManager->addSuccessMessage(
                         __(
                             "The integration '%1' has been deleted.",
                             $this->escaper->escapeHtml($integrationData[Info::DATA_NAME])
```

Comment:
addSuccessMessage works a little bit differently way then addSuccess method. By default, it escapes all HTML. So, please check that message is correct and also you can remove  additional escape `$this->escaper->escapeHtml`